openwisp_monitoring/device/static/monitoring/js/alert-settings.js

```diff
@@ -0,0 +1,11 @@
+(function ($) {
+    'use strict';
+    $(function () {
+        var checks = $('[id^="id_monitoring-metric-content_type-object_id"][id $="alertsettings-0-is_active"]');
```

Comment:
I don't recall why I named it `checks`. Maybe we should rename it (something like `is_active` simply), since `checks` can create confusion that it's related to the `Check` model. Please feel free to rename it, if you think it's a better name.